MODULE.bazel

```diff
@@ -5,7 +5,7 @@ module(
     repo_name = "rules_xcodeproj",
 )
 
-bazel_dep(name = "bazel_skylib", version = "1.3.0")
```

Comment:
I don't like forcing this on users. https://bazelbuild.slack.com/archives/C014RARENH0/p1691156601420349